Appcenter build fails

The Appcenter Builds still fail: https://appcenter.ms/orgs/ywesee-GmbH/apps/AmiKo-CoMed/build/branches/master/builds/7

See attached log.
[8_Restore Nuget.txt](https://github.com/zdavatz/amiko_csharp/files/3278091/8_Restore.Nuget.txt)

Error message is `##[error]The nuget command failed with exit code(1) and error(Errors in packages.config projects`

--- a/NuGet.config
+++ b/NuGet.config
@@ -3,4 +3,7 @@
   <config>
     <add key="repositoryPath" value="Packages" />
   </config>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
 </configuration>

